albumentations/augmentations/transforms.py

```diff
@@ -605,7 +606,104 @@ def get_transform_init_args_names(self):
         return ("mean", "std", "max_pixel_value")
 
 
-class Cutout(ImageOnlyTransform):
+class RandomErasing(DualTransform):
+    """ Randomly selects a rectangle region in an image and erases its pixels.
+        'Random Erasing Data Augmentation' by Zhong et al.
+        See https://arxiv.org/pdf/1708.04896.pdf
+    Args:
+         probability: The probability that the Random Erasing operation will be performed.
+         sl: Minimum proportion of erased area against input image.
+         sh: Maximum proportion of erased area against input image.
+         r1: Minimum aspect ratio of erased area.
+         num_holes: Number of erasing areas
+         mean: Erasing value.
```

Comment:
`__init__` doesn't contain `mean` parameter

Reply:
The doc string is somehow not aligned. will be fixed.